qemu/tests/cfg/live_snapshot.cfg

```diff
@@ -95,6 +95,27 @@
                     reboot_method = "system_reset"
                     when_start = "reboot"
                     after_finished = "verify_alive"
+                - data_integrity:
+                    type = live_snapshot_integrity
+                    drive_cache = writethrough
+                    before_start = "load_stress"
+                    after_finished = "unload_stress"
+                    Windows:
+                        create_command = "D:\coreutils\DummyCMD.exe FILE 1073741824 1"
+                        file_names = "C:\live_snapshot_base C:\live_snapshot_sn1"
```

Comment:
Is that the name `live_snapshot_base` means this file would be created in the base image? If so, it is better to do the clean-up for it.

Reply:
@luckyh Thanks, since it will backup the whole guest image before test and then recover with backup after test finished, so I suppose that clean up inside guest is not necessary.